daemon/src/common/String.h

```diff
@@ -241,6 +242,18 @@ namespace Str {
             return c;
     }
 
+    /*
+     * Converts a hexadecimal character to the value of the digit it represents.
+     * Pre: cisxdigit(ch)
+     */
+    inline CONSTEXPR_FUNCTION int GetHex(char ch)
```

Comment:
What benefit would come from this being constexpr? If there are no benefits, I'd rather have this inlined at LTO.